front-end/src/Hamburger.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { slide as Menu } from 'react-burger-menu'
+
+const props = () => {
```

Comment:
Looks good! The only thing that confuses me is this and `export default props`

Would it be possible to rework this component to be more like this:
`const Hamburger = (props) => {` and then export Hamburger?

Reply:
Made those changes.